### DISCUSSION TITLE
I can't get the extension to run, I get this error

### DISCUSSION THREAD

**Discussion by Datatrax** [innightwolfsleep/llm_telegram_bot#151] [Q&A] [answered]
![image](https://github.com/innightwolfsleep/text-generation-webui-telegram_bot/assets/72715119/ebb816ed-3fdd-4011-8bfa-a8fe24efc622)

**Comment by innightwolfsleep** [comment]
Try to run `cmd_windows.bat`
and `pip install -r text-generation-webui\extensions\telegram_bot\requirements_ext.txt`

**Reply by Datatrax** [reply]
thank you for replying, when I do this happens. I am on Win11

![image](https://github.com/innightwolfsleep/text-generation-webui-telegram_bot/assets/72715119/4cbf3b36-de0d-45ef-a8e9-8c8d55bae922)

**Reply by Datatrax** [reply]
![image](https://github.com/innightwolfsleep/text-generation-webui-telegram_bot/assets/72715119/e36d5794-ebd8-4886-9168-1acdba110774)

**Reply by Datatrax** [reply]
when I do it with the normal console I get this

![image](https://github.com/innightwolfsleep/text-generation-webui-telegram_bot/assets/72715119/6a600d51-1c90-43b5-88a7-65d4ba369ebe)

but if i do with cmd_windows.bat it: "not such file or directory"

I also have these files in the python path:
![image](https://github.com/innightwolfsleep/text-generation-webui-telegram_bot/assets/72715119/f2170eab-5ab2-4018-92f5-78371532a964)

**Comment by innightwolfsleep** [comment, marked as answer]
![image](https://github.com/innightwolfsleep/text-generation-webui-telegram_bot/assets/20921671/b310c91b-6858-410f-86c6-240f95539085)
It is work for me. Perhaps, there is changes in file structure of new oobabooga version. Check it later... 
But I think there is wrong path to `requirements_ext.txt` file in pip install command. Perhaps, 
```
pip install -r extensions\telegram_bot\requirements_ext.txt
```

**Reply by Datatrax** [reply]
It works now. Thank you very much!
